src/views-components/details-panel/details-panel.tsx

```diff
@@ -0,0 +1,145 @@
+// Copyright (C) The Arvados Authors. All rights reserved.
+//
+// SPDX-License-Identifier: AGPL-3.0
+
+import * as React from 'react';
+import Drawer from '@material-ui/core/Drawer';
+import IconButton from "@material-ui/core/IconButton";
+import CloseIcon from '@material-ui/icons/Close';
+import { StyleRulesCallback, WithStyles, withStyles, Theme } from "@material-ui/core/styles";
+import Tabs from '@material-ui/core/Tabs';
+import Tab from '@material-ui/core/Tab';
+import Typography from '@material-ui/core/Typography';
+import * as classnames from "classnames";
+import Grid from '@material-ui/core/Grid';
+
+function TabContainer(props: any) {
```

Comment:
You should use `const TabContainer: React.SFC = props => <Typography>{props.children}</Typography>` to avoid using `any` as a props type.